unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -520,13 +602,7 @@ public void GoToMarketplace()
             WebInterface.OpenURL("https://docs.decentraland.org/get-a-wallet");
     }
 
-    public void SellCollectible(string collectibleId)
-    {
-        WebInterface.OpenURL("https://market.decentraland.org/account");
-    }
+    public void SellCollectible(string collectibleId) { WebInterface.OpenURL("https://market.decentraland.org/account"); }
```

Comment:
can we move this to a constant?

Reply:
Of course! Done in https://github.com/decentraland/explorer/pull/2300/commits/f8e457358c9548fb2836f6640f4efac966f39fd0. Thanks!